File: PRAK602-2210817210023-Muhammad Shofy Akmal.py
kl = int(input())
isi = list(map(int, input().split()))
a = []
for b in range(0, kl) :
    a.append(isi[b])
print ()
for b in range(0, kl) :
    print(a[b]*(b+1), end=' ')

File: PRAK602-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
int main () {
int kl, isi, a;
scanf ("%d", &kl);
int b[kl];
for (a=0; a<kl; a++) {
    scanf ("%d", &isi);
    b[a]=isi;}
printf ("\n");
for (a=0; a<kl; a++) {
    printf ("%d ", b[a]*(a+1));}
}